py/kubeflow/testing/cleanup_ci.py

```diff
@@ -17,19 +17,26 @@
 from googleapiclient import discovery
 from oauth2client.client import GoogleCredentials
 
+# See https://github.com/kubeflow/testing/issues/444
+# We are switching to unique names for auto deployments
+# So this matches the new ones.
+AUTO_DEPLOY_PATTERN = re.compile("kf-vmaster-(?!n\d\d)")
```

Comment:
this only matches auto-deployment from master?  how about current stable release?

Reply:
This is deliberate. The existing auto deployed clusters aren't cleaned up by cleanup_ci.py. Instead the auto deployment script recycles clusters as it cycles through names.